<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MagneticUi</class>
 <widget class="QWidget" name="MagneticUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QLabel#current_x, #current_y, #current_z, #offset_x, #offset_y, #offset_z {
background-color: #FFFFFF;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #000000;
text-align: center;
border: 1px solid #007AFF;
}

QFrame#frame_current_values, #frame_offset {
background-color:#F0F0F0;
border: 1px solid #CCCCCC;
}
QPushButton:enabled {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #FFFFFF;
text-align: center;
}

QPushButton:disabled{
background-color: #7AB7FA;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #FFFFFF;
text-align: center;
}

QPushButton#pb_back {
background-color: transparent;
border:none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>440</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="pb_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/ui/images/ic_back.png</normaloff>:/ui/images/ic_back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family: SourceHanSansCN-Normal;
font-size: 18px;
color: #000000;
text-align: right;
}</string>
       </property>
       <property name="text">
        <string>椭球拟合法</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Ellipse" name="frame_xy"/>
     </item>
     <item>
      <widget class="Ellipse" name="frame_yz"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
     <item>
      <widget class="Ellipse" name="frame_xz"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_current_values">
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1,1">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>28</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>当前值</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>X:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="current_x">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="current_y">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>Z:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="current_z">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_offset">
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,1">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>28</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>补偿</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="offset_x">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="offset_y">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>Z:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="offset_z">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>7</number>
           </property>
           <item alignment="Qt::AlignLeft">
            <widget class="QPushButton" name="pb_guide">
             <property name="styleSheet">
              <string notr="true">QPushButton#pb_guide {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #0B22C6;
text-align: center;
text-decoration:underline;
}</string>
             </property>
             <property name="text">
              <string>操作说明</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>30</number>
             </property>
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="pb_start">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>开始校准</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_finish">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pb_guide {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #0B22C6;
text-align: center;
text-decoration:underline;
}</string>
               </property>
               <property name="text">
                <string>保存</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ellipse</class>
   <extends>QFrame</extends>
   <header>ellipse.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
